pkg/scoring/condition_http_test.go

```
@@ -223,3 +223,129 @@ func TestNegatedLeaf_ArrayLengthGte_GivesLessThan(t *testing.T) {
 	assert.True(t, fired, "2 scopes is fewer than 25, so the negated gte should fire")
 }
+
+// ----------------------------------------------------------------
+// cache key must reflect the RENDERED request, not the template (LAB-6051)
+// ----------------------------------------------------------------
+
+// countingServer records every request path/header/body it receives.
+type countingServer struct {
+	calls   int
+	paths   []string
+	regions []string
```

golangci-lint (2.12.2)

[error] field regions is unused

(unused)